fixes #2521

benemon opened the pull request:
Small change to address issue #2521

alexkieling commented on hawtio-system/src/main/java/io/hawt/jmx/JmxTreeWatcher.java:
```diff
@@ -61,7 +61,7 @@ public void init() throws Exception {
             if (Strings.isNotBlank(text)) {
```
I think you can remove this `if` statement.

benemon replied:
To be fair, the whole thing can now probably just be boiled down to `LOG.info("Welcome to hawtio " +  getVersion());`

tadayosi replied:
Use

    LOG.info("Welcome to hawtio {}", getVersion());

instead of

    LOG.info("Welcome to hawtio " + getVersion());

tadayosi commented:
Also we now follow this commit message guideline: https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines
So should the commit as well. Squashing them is also preferable for such small changes.

benemon commented:
> Also we now follow this commit message guideline: https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines

Would be good to have that in the Contribution docs.

I've squashed the commits and updated the log message as requested.

jamesnetherton commented:
With `{}` placeholders we don't need to use string concatenation. So it should look like:

`LOG.info("Welcome to hawtio {}", getVersion());`

If you can make that correction, we can get this merged.

benemon commented:
Sorry, I misread @tadayosi 's comment as providing a delimiter between the message and the version.

Updated and re-pushed.
